llvm/tools/llvm-objcopy/ObjcopyOpts.td

```diff
@@ -205,28 +204,45 @@ defm prefix_symbols
       MetaVarName<"prefix">;
 
 defm prefix_alloc_sections
-    : Eq<"prefix-alloc-sections", "Add <prefix> to the start of every allocated section name">,
+    : Eq<"prefix-alloc-sections",
+         "Add <prefix> to the start of every allocated section name">,
       MetaVarName<"prefix">;
 
-defm set_start : Eq<"set-start", "Set the start address to <addr>. Overrides "
-                    "any previous --change-start or --adjust-start values">,
-                 MetaVarName<"addr">;
-defm change_start : Eq<"change-start", "Add <incr> to the start address. Can be "                        
-                       "specified multiple times, all values will be applied "
-                       "cumulatively">,
-                    MetaVarName<"incr">;
+defm set_start
+    : Eq<"set-start", "Set the start address to <addr>. Overrides "
+                      "any previous --change-start or --adjust-start values">,
+      MetaVarName<"addr">;
+defm change_start
+    : Eq<"change-start", "Add <incr> to the start address. Can be "
+                         "specified multiple times, all values will be applied "
+                         "cumulatively">,
+      MetaVarName<"incr">;
 def adjust_start : JoinedOrSeparate<["--"], "adjust-start">,
                    Alias<change_start>,
                    HelpText<"Alias for --change-start">;
 
 defm add_symbol
-    : Eq<"add-symbol", "Add new symbol <name> to .symtab. Accepted flags: "
-         "global, local, weak, default, hidden, protected, file, section, object, "
+    : Eq<"add-symbol",
+         "Add new symbol <name> to .symtab. Accepted flags: "
+         "global, local, weak, default, hidden, protected, file, section, "
+         "object, "
          "function, indirect-function. Accepted but ignored for "
          "compatibility: debug, constructor, warning, indirect, synthetic, "
          "unique-object, before">,
       MetaVarName<"name=[section:]value[,flags]">;
 
 defm update_section
-    : Eq<"update-section", "Replace the contents of section <name> with contents from a file <file>">,
+    : Eq<"update-section", "Replace the contents of section <name> with "
+                           "contents from a file <file>">,
       MetaVarName<"name=file">;
+
+defm adjust_section_vma
```

Comment:
I think it makes sense to be able to specify this option multiple times? Also, it might make sense to add the --change-addresses/--change-section-address options at the same time (I think they're just aliases from the GNU objcopy docs, but I haven't experimented with them).